sale_invoice_policy/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale invoice Policy",
+    "summary": """
+        Sales Management: let the user choose the invoice policy on the
+        order""",
+    "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sales Management",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["sale", "sale_stock"],
+    "data": [
+        "data/sale_data.xml",
+        "wizards/res_config_view.xml",
+        "views/product_template_view.xml",
+        "views/sale_view.xml",
+    ],
+    "pre_init_hook": "pre_init_hook",
```

Comment:
should stay in post_init_hook as in pre-state "invoice_policy" field will not de recomputed with code presented in this addon

Reply:
I do not understand the problem. The _pre_init_hook()_ just creates in SQL the new _default_invoice_policy_ column and initializes it with the same value as _invoice_policy_.
It does not need any python code at all.

Reply:
nevermind, you are right